handler_test.go

```diff
@@ -581,7 +581,7 @@ func TestSession(t *testing.T) {
 	r.Use(authcontrol.Session(authOptions))
 	r.Use(authcontrol.AccessControl(ACL, authOptions))
 	r.Use(middleware.VerifyQuota(client, quotaOptions))
-	r.Use(middleware.RateLimit(cfg.RateLimiter, limitCounter, quotaOptions))
+	r.Use(middleware.RateLimit(client, cfg.RateLimiter, limitCounter, quotaOptions))
```

Comment:
Instead of breaking the `.RateLimit()` API, I think I'd prefer to use a different `Prefix` on the `limitCounter`.

the prefix has one important benefit -- it's easier to debug from within Redis CLI/UI
as they key will be for example `"qc:rate:indexer:{number}"`
and we can do stuff like count of `qc:rate:*` vs. `qc:rate:indexer:*` vs. `qc:rate:metadata:*`

Reply:
That sounds good, we will still be breaking an interface, because we need to pass an extra argument here:
https://github.com/0xsequence/quotacontrol/blob/d3804b4b7d81b31c5c9f0e4248c7456590f8eed4/cache.go#L56